src/libraries/System.Runtime/tests/System/CharTests.cs

```diff
@@ -199,6 +199,15 @@ public void GetTypeCode_Invoke_ReturnsBoolean()
             Assert.Equal(TypeCode.Char, 'a'.GetTypeCode());
         }
 
+        [Fact]
+        public static void IsAscii_Char()
+        {
+            Assert.True(char.IsAscii(char.MinValue));
+            Assert.True(char.IsAscii('\x007f'));
+            Assert.False(char.IsAscii('\x0080'));
+            Assert.False(char.IsAscii(char.MaxValue));
```

Comment:
Is this enough for testing, or should be use the "whole range" ala https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Runtime/tests/System/Text/RuneTests.TestData.cs#L233
?

Reply:
The whole range. There is probably a similar test for Rune.IsAscii

Reply:
Just scanned again through the repo, looking for IsAscii-tests. Haven't found one.

I've seen that all IsAscii implementations, like https://github.com/dotnet/runtime/blob/aa5fdab9654d74bc6274c0b5d820272c8e859621/src/libraries/System.Private.CoreLib/src/System/Text/Rune.cs#L143 delegate to https://github.com/dotnet/runtime/blob/6072e4d3a7a2a1493f514cdf4be75a3d56580e84/src/libraries/System.Private.CoreLib/src/System/Text/UnicodeUtility.cs#L120

It's a trivial method (just one compare), but maybe it's better to delegate `char.IsAscii` also to this method and add tests over there (as I didn't find some for `IsAsciiCodePoint`)?

Reply:
Search the file https://github.com/dotnet/runtime/blob/master/src/libraries/System.Runtime/tests/System/Text/RuneTests.cs for the substring `"IsAscii"`. You'll see how we perform tests against the `Rune` APIs.

Though to be honest those tests are basically just testing specific values like `'\u0000'`, `'\u007f'`, etc., as you had here. :)

I have no problem with you leaving the test as you have it here.